data/json/items/ammo.json

```diff
@@ -1503,6 +1503,258 @@
+    "color": "black",
+    "looks_like": "marble",
+    "description": "A loose black knight from a game of chess.  This one is designed to resemble a horse’s head and neck.",
+    "material": [ "wood" ],
+    "volume": "15 ml",
+    "weight": "30 g",
+    "ammo_type": "pebble",
+    "damage": { "damage_type": "bullet", "amount": 1 },
+    "range": 20,
+    "dispersion": 15,
+    "loudness": 0,
+    "count": 7,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "type": "AMMO",
+    "id": "bchess_bishop",
+    "price": 10,
+    "price_postapoc": 10,
+    "name": { "str": "black bishop" },
+    "symbol": "=",
+    "color": "black",
+    "looks_like": "marble",
+    "description": "A loose black bishop from a game of chess.  This one is represented by a stylized miter.",
+    "material": [ "wood" ],
+    "volume": "15 ml",
+    "weight": "30 g",
+    "ammo_type": "pebble",
+    "damage": { "damage_type": "bullet", "amount": 1 },
+    "range": 20,
+    "dispersion": 15,
+    "loudness": 0,
+    "count": 7,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "type": "AMMO",
+    "id": "bchess_rook",
+    "price": 10,
+    "price_postapoc": 10,
+    "name": { "str": "black rook" },
+    "symbol": "=",
+    "color": "black",
+    "looks_like": "marble",
+    "description": "A loose black rook from a game of chess.  This one is designed to resemble a castle tower with crenellations on top.",
+    "material": [ "wood" ],
+    "volume": "15 ml",
+    "weight": "30 g",
+    "ammo_type": "pebble",
+    "damage": { "damage_type": "bullet", "amount": 1 },
+    "range": 20,
+    "dispersion": 15,
+    "loudness": 0,
+    "count": 7,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "type": "AMMO",
+    "id": "bchess_queen",
+    "price": 10,
+    "price_postapoc": 10,
+    "name": { "str": "black queen" },
+    "symbol": "=",
+    "color": "black",
+    "looks_like": "marble",
+    "description": "A loose black queen from a game of chess.  It is the second tallest piece and this one is represented by a coronet.",
+    "material": [ "wood" ],
+    "volume": "15 ml",
+    "weight": "30 g",
+    "ammo_type": "pebble",
+    "damage": { "damage_type": "bullet", "amount": 1 },
+    "range": 20,
+    "dispersion": 15,
+    "loudness": 0,
+    "count": 7,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "type": "AMMO",
+    "id": "bchess_king",
+    "price": 10,
+    "price_postapoc": 10,
+    "name": { "str": "black king", "str_pl": "black kings" },
+    "symbol": "=",
+    "color": "black",
+    "looks_like": "marble",
+    "description": "A loose black king from a game of chess.  It is the tallest piece in the game, this one is represented by a cross on the top.",
+    "material": [ "wood" ],
+    "volume": "15 ml",
+    "weight": "30 g",
+    "ammo_type": "pebble",
+    "damage": { "damage_type": "bullet", "amount": 1 },
+    "range": 20,
+    "dispersion": 15,
+    "loudness": 0,
+    "count": 7,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
```

Comment:
There is no reason to have it as ammo, throwing wooden stuff onto enemies is never justified (and 1 bullet damage is too high for this)
Additionally, simplified the syntax using copy-from and bring dimensions to official FIDE chess sets

```suggestion
  {
    "type": "GENERIC",
    "id": "wchess_pawn",
    "name": { "str": "white pawn" },
    "description": "A loose white pawn from a game of chess.  It is the smallest piece and this one is represented by a simple, round-headed shape.",
    "price": 60,
    "price_postapoc": 1,
    "symbol": "=",
    "color": "white",
    "looks_like": "marble",
    "material": [ "wood" ],
    "volume": "24 ml",
    "weight": "20 g"
  },
  {
    "type": "GENERIC",
    "id": "wchess_knight",
    "copy-from": "wchess_pawn",
    "name": { "str": "white knight" },
    "symbol": "=",
    "color": "white",
    "description": "A loose white knight from a game of chess.  This one is designed to resemble a horse’s head and neck.",
    "volume": "38 ml",
    "weight": "32 g"
  },
  {
    "type": "GENERIC",
    "id": "wchess_bishop",
    "copy-from": "wchess_pawn",
    "name": { "str": "white bishop" },
    "symbol": "=",
    "color": "white",
    "description": "A loose white bishop from a game of chess.  This one is represented by a stylized miter.",
    "volume": "44 ml",
    "weight": "37 g"
  },
  {
    "type": "GENERIC",
    "id": "wchess_rook",
    "copy-from": "wchess_pawn",
    "name": { "str": "white rook" },
    "symbol": "=",
    "color": "white",
    "description": "A loose white rook from a game of chess.  This one is designed to resemble a castle tower with crenellations on top.",
    "volume": "40 ml",
    "weight": "34 g"
  },
  {
    "type": "GENERIC",
    "id": "wchess_queen",
    "copy-from": "wchess_pawn",
    "name": { "str": "white queen" },
    "symbol": "=",
    "color": "white",
    "description": "A loose white queen from a game of chess.  It is the second tallest piece and this one is represented by a coronet.",
    "volume": "58 ml",
    "weight": "49 g"
  },
  {
    "type": "GENERIC",
    "id": "wchess_king",
    "copy-from": "wchess_pawn",
    "name": { "str": "white king" },
    "symbol": "=",
    "color": "white",
    "description": "A loose white king from a game of chess.  It is the tallest piece in the game, this one is represented by a cross on the top.",
    "volume": "68 ml",
    "weight": "58 g"
  },
  {
    "type": "GENERIC",
    "id": "bchess_pawn",
    "copy-from": "wchess_pawn",
    "name": { "str": "black pawn" },
    "symbol": "=",
    "color": "black",
    "description": "A loose black pawn from a game of chess.  It is the smallest piece and this one is represented by a simple, round-headed shape.",
    "volume": "24 ml",
    "weight": "20 g"
  },
  {
    "type": "GENERIC",
    "id": "bchess_knight",
    "copy-from": "wchess_pawn",
    "name": { "str": "black knight" },
    "symbol": "=",
    "color": "black",
    "description": "A loose black knight from a game of chess.  This one is designed to resemble a horse’s head and neck.",
    "volume": "38 ml",
    "weight": "32 g"
  },
  {
    "type": "GENERIC",
    "id": "bchess_bishop",
    "copy-from": "wchess_pawn",
    "name": { "str": "black bishop" },
    "symbol": "=",
    "color": "black",
    "description": "A loose black bishop from a game of chess.  This one is represented by a stylized miter.",
    "volume": "44 ml",
    "weight": "37 g"
  },
  {
    "type": "GENERIC",
    "id": "bchess_rook",
    "copy-from": "wchess_pawn",
    "name": { "str": "black rook" },
    "symbol": "=",
    "color": "black",
    "description": "A loose black rook from a game of chess.  This one is designed to resemble a castle tower with crenellations on top.",
    "volume": "40 ml",
    "weight": "34 g"
  },
  {
    "type": "GENERIC",
    "id": "bchess_queen",
    "copy-from": "wchess_pawn",
    "name": { "str": "black queen" },
    "symbol": "=",
    "color": "black",
    "description": "A loose black queen from a game of chess.  It is the second tallest piece and this one is represented by a coronet.",
    "volume": "58 ml",
    "weight": "49 g"
  },
  {
    "type": "GENERIC",
    "id": "bchess_king",
    "copy-from": "wchess_pawn",
    "name": { "str": "black king", "str_pl": "black kings" },
    "symbol": "=",
    "color": "black",
    "description": "A loose black king from a game of chess.  It is the tallest piece in the game, this one is represented by a cross on the top.",
    "volume": "68 ml",
    "weight": "58 g"
  },
```